codegen/src/main/resources/templates/query/ModelStaxUnmarshaller.ftl

```diff
@@ -141,9 +151,8 @@ public class ${shape.shapeName}Unmarshaller implements Unmarshaller<${shape.shap
             } else if (xmlEvent.isEndElement()) {
                 if (context.getCurrentDepth() < originalDepth) {
 <#list shape.members as memberModel>
-    <#if memberModel.map && (!memberModel.http.location?? || memberModel.http.location == "headers")>
-                    ${shape.variable.variableName}.${memberModel.fluentSetterMethodName}(${memberModel.variable.variableName});
-
+    <#if (memberModel.map || memberModel.list) && (!memberModel.http.location?? || memberModel.http.location == "headers")>
```

Comment:
Lists previously were appended to by calling the varargs setter. Maps were appended to by aggregating the results and then saving them at the end.

Vargs setters no longer append, so I just changed lists to work like maps. I'll see if I can figure out how to clean up the boolean condition. It'll give me time to learn more about freemarker before we stop using it!